### DISCUSSION TITLE
Which create/update field should we use for a n - to - n relationship?

### DISCUSSION THREAD

**Discussion by realtebo** [Laravel-Backpack/community-forum#569] [Q&A (Help)] [answered]
We have a school, so there is a n-to-n relationship between users and institute classes ("3H, 4I, 1C, ... etc..");

When editing a teacher I'd like a fast way for user to activate/deactivate the association with classes. 

Is there a field handling many-to-many relationship in a list-like style ? (list of radios, I think)

Actually I'm using the super magic select2 served by `type('relationship')`. But I think a list of radios will be simpler to use

        CRUD::field('teacher_classes')
            ->label("classes")
            ->type('relationship')
            ->attribute('name')
            ->placeholder("Select all teacher's classes")
        ;

**Comment by tabacitu** [comment, marked as answer]
> Is there a field handling many-to-many relationship in a list-like style ? (list of radios, I think)

Nope.
